src/CommunityToolkit.Maui/Animations/FadeAnimation.shared.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace CommunityToolkit.Maui.Animations;
+
+/// <summary>
+/// Animation that will fade the supplied view to the specified <see cref="Opacity"/>
+/// and then back to it's original <see cref="Opacity"/>.
+/// </summary>
+public class FadeAnimation : BaseAnimation
+{
+	/// <summary>
+	/// Initializes a new instance of <see cref="FadeAnimation"/>.
+	/// </summary>
+	public FadeAnimation() : base(300)
```

Comment:
what is 300 here? is it duration?

Reply:
It's the default length of the animation: https://github.com/CommunityToolkit/Maui/blob/5ade8c29f906e08235fe5996b6d8d173ff74ac36/src/CommunityToolkit.Maui/Animations/BaseAnimation.shared.cs#L61